scalatest-test/src/test/scala/org/scalatest/prop/RandomizerSpec.scala

```diff
@@ -53,38 +53,6 @@ class RandomizerSpec extends FunSpec with Matchers {
       jc shouldEqual ic
     }
 
-    it("should offer a nextFloatBetween0And1 method that produces the same values as java.util.Random given the same seed") {
```

Comment:
@bvenners Since the old tests are being removed, I think it is good to add back a test that make sure the method returns the value between 0.0-1.0, e.g. may just generate 10 numbers and make sure they are in the range.